sys/arm/nvidia/tegra_pcie.c

```diff
@@ -835,17 +835,16 @@ tegra_pcib_msi_map_msi(device_t dev, device_t child, struct intr_irqsrc *isrc,
 static bus_size_t
 tegra_pcib_pex_ctrl(struct tegra_pcib_softc *sc, int port)
 {
-	if (port >= TEGRA_PCIB_MAX_PORTS)
-		panic("invalid port number: %d\n", port);
-
-	if (port == 0)
-		return (AFI_PEX0_CTRL);
-	else if (port == 1)
-		return (AFI_PEX1_CTRL);
-	else if (port == 2)
-		return (AFI_PEX2_CTRL);
-	else
-		panic("invalid port number: %d\n", port);
+	switch (port) {
+	case 0:
+                return (AFI_PEX0_CTRL);
+	case 1:
+                return (AFI_PEX1_CTRL);
+	case 2:
+                return (AFI_PEX_2_CTRL);
```

Comment:
Some undesired whitespace change slipped through - tabs are replaced by spaces in all "return" + "panic" lines.